### ISSUE TITLE
make_mth5_driver_v0.2.0.ipynb not running to completion

### ISSUE THREAD

**Issue by kkappler** [kujaku11/mth5#147]
On master branch of mth5 and main of mt_metadata, this notebook gets hung up on cell # 23.

Perhaps we should think about strategies discussed [here](https://mg.readthedocs.io/git-jupyter.html) for running ipynbs, maybe only when merging into main/master or when bump version or something?

The code block that fails is:
```
for tf_fn in [cas04_tf, nvr08_tf]:
    tf_obj = TF(tf_fn)
    tf_obj.read_tf_file()
    mth5_object.add_transfer_function(tf_obj)
```
with error:
`ValueError: The input run id is None. Input a string or integer`

**Comment by kkappler** [comment]
Re-running this it is now failing earlier -- on cell 14:
```
ex = mth5_object.from_reference(ch_df.iloc[0].hdf5_reference).to_channel_ts()
print(ex)
```
yields:

> ValueError                                Traceback (most recent call last)
/tmp/ipykernel_22725/857314163.py in <module>
----> 1 mth5_object.from_reference(ch_df.iloc[0].hdf5_reference).to_channel_ts()
~/software/irismt/mth5/mth5/groups/channel_dataset.py in to_channel_ts(self)
    676             station_metadata=self.station_metadata.copy(),
    677             survey_metadata=self.survey_metadata.copy(),
--> 678             channel_response_filter=self.channel_response_filter,
    679         )
    680 
~/software/irismt/mth5/mth5/timeseries/channel_ts.py in __init__(self, channel_type, data, channel_metadata, station_metadata, run_metadata, survey_metadata, **kwargs)
    123         self.station_metadata = station_metadata
    124         self.run_metadata = run_metadata
--> 125         self.channel_metadata = channel_metadata
    126 
    127         # input data
~/software/irismt/mth5/mth5/timeseries/channel_ts.py in channel_metadata(self, channel_metadata)
    567                 self.channel_type = self.run_metadata.channels[0].type
    568             else:
--> 569                 raise ValueError("Channel ID cannot be None")
    570 
    571     @property
ValueError: Channel ID cannot be None

**Comment by kkappler** [comment]
I committed an duplicate notebook with no outputs on `fix_issue_147` branch.

This was not running on push, and I'm not sure why, so I exported it to a .py file and it runs, 
and give the expected error message.

**Comment by kujaku11** [comment]
Fixed with PR #150
